client/src/api/axiosClient.js

```diff
@@ -0,0 +1,41 @@
+import axios from "axios";
+import queryString from "query-string";
+
+const axiosClient = axios.create({
+  baseURL: "",
+  headers: {
+    "content-type": "application/json",
+  },
+  paramsSerializer: (params) => queryString.stringify(params),
+});
+
+axiosClient.interceptors.request.use(async (config) => {
+  // Handle token here...
+  const currentUser = localStorage.getItem("userInfo")
+    ? JSON.parse(localStorage.getItem("userInfo"))
+    : null;
```

Comment:
It looks like Jr is actually using `loggedIn`
[code here](https://github.com/hatchways/team-kim-possible/pull/41/files#diff-d0043ff4661bea3626e36462596c4c8c7ca4c0e79622a54112669df13943c4deR72)